include/exadg/aero_acoustic/driver.cpp

```diff
@@ -214,14 +214,14 @@ Driver<dim, Number>::print_performance_results(double const total_time) const
 
   pcout << std::endl << "Throughput related to one macro time step:";
   print_throughput_unsteady(pcout,
-                            (double)DoFs_f + sub_dt_per_macro_dt * (double)DoFs_a,
+                            DoFs_f + DoFs_a,
                             time_solvers_side_by_side_avg,
                             acoustic->get_number_of_macro_time_steps(),
                             N_mpi_processes);
 
   pcout << std::endl << "Throughput related to one sub time step:";
   print_throughput_unsteady(pcout,
-                            DoFs_f + DoFs_a,
+                            (double)DoFs_f / sub_dt_per_macro_dt + (double)DoFs_a,
                             time_solvers_side_by_side_avg,
```

Comment:
I just had a brief look again at this PR. I have given two equivalent formulas in comment https://github.com/exadg/exadg/pull/645#discussion_r1562664109 for the throughput per time step, and I currently believe if we compute the dofs like this, we have to divide the wall time by the number of substeps. Alternatively, use the equation for the number of dofs which you have deleted above in line 217 in the last commit 6a024cd4c59a5c7d5e2266b34be6adc6e621e569, or use the old code and just switch the output "macro time step" and "sub time step" in the `pcout` commands.